### DISCUSSION TITLE
2020/using-wireshark-remote-capture-with-eve-ng/

### DISCUSSION THREAD

**Discussion by giscus** [hellt/netdevops.me#22] [Show and tell]
# 2020/using-wireshark-remote-capture-with-eve-ng/

Applying DevOps to networks.

https://netdevops.me/2020/using-wireshark-remote-capture-with-eve-ng/

**Comment by zalbarqa** [comment]
what should i type in the remote ssh server address ?
- am using pnetlab its a platform same as eve-ng
- am getting below log in the right corner while capturing:
capture://192.168.4.201/vunl90_0
